gateway/gateway.go

```diff
@@ -237,6 +238,11 @@ func (gateway *HandleT) dbWriterWorkerProcess(process int) {
 		} else {
 			gateway.jobsDB.Store(jobList)
 		}
+		batchPayloadSize := 0
+		for _, req := range jobList {
+			batchPayloadSize += len(req.EventPayload)
+		}
+		gateway.batchPayloadSizeStat.Count(batchPayloadSize)
```

Comment:
Here we are measuring DB write size, instead, let's measure the event size. 

Either size of the HTTP request payload or single event size. Once we have that, we can derive these values using some heuristics.

Reply:
done